apartment_application_service/settings.py

```diff
@@ -329,11 +333,15 @@
     },
 }
 
-SAP_SFTP_USERNAME = env("SAP_SFTP_USERNAME")
-SAP_SFTP_PASSWORD = env("SAP_SFTP_PASSWORD")
-SAP_SFTP_HOST = env("SAP_SFTP_HOST")
-SAP_SFTP_PORT = env("SAP_SFTP_PORT")
-SAP_SFTP_FILENAME_PREFIX = env("SAP_SFTP_FILENAME_PREFIX")
+SAP_SEND_SFTP_USERNAME = env("SAP_SEND_SFTP_USERNAME")
+SAP_SEND_SFTP_PASSWORD = env("SAP_SEND_SFTP_PASSWORD")
+SAP_SEND_SFTP_HOST = env("SAP_SEND_SFTP_HOST")
+SAP_SEND_SFTP_PORT = env("SAP_SEND_SFTP_PORT")
+SAP_SEND_SFTP_FILENAME_PREFIX = env("SAP_SEND_SFTP_FILENAME_PREFIX")
+SAP_FETCH_SFTP_USERNAME = env("SAP_FETCH_SFTP_USERNAME")
+SAP_FETCH_SFTP_PASSWORD = env("SAP_FETCH_SFTP_PASSWORD")
+SAP_FETCH_SFTP_HOST = env("SAP_FETCH_SFTP_HOST")
+SAP_FETCH_SFTP_PORT = env("SAP_FETCH_SFTP_PORT")
```

Comment:
It would be best to keep it compatible with the old env variables for sending:
```suggestion
SAP_SFTP_USERNAME = env("SAP_SFTP_USERNAME")
SAP_SFTP_PASSWORD = env("SAP_SFTP_PASSWORD")
SAP_SFTP_HOST = env("SAP_SFTP_HOST")
SAP_SFTP_PORT = env("SAP_SFTP_PORT")
SAP_SFTP_FILENAME_PREFIX = env("SAP_SFTP_FILENAME_PREFIX")

SAP_SFTP_SEND_USERNAME = env("SAP_SEND_SFTP_USERNAME", default=SAP_SFTP_USERNAME)
SAP_SFTP_SEND_PASSWORD = env("SAP_SEND_SFTP_PASSWORD", default=SAP_SFTP_PASSWORD)
SAP_SFTP_SEND_HOST = env("SAP_SEND_SFTP_HOST", default=SAP_SFTP_HOST)
SAP_SFTP_SEND_PORT = env("SAP_SEND_SFTP_PORT", default=SAP_SFTP_PORT)
SAP_SFTP_SEND_FILENAME_PREFIX = env(
    "SAP_SFTP_SEND_FILENAME_PREFIX", default=SAP_SFTP_FILENAME_PREFIX
)

SAP_SFTP_FETCH_USERNAME = env("SAP_SFTP_FETCH_USERNAME")
SAP_SFTP_FETCH_SFTP_PASSWORD = env("SAP_SFTP_FETCH_PASSWORD")
SAP_SFTP_FETCH_SFTP_HOST = env("SAP_SFTP_FETCH_HOST")
SAP_SFTP_FETCH_SFTP_PORT = env("SAP_SFTP_FETCH_PORT")
```